test/404s.test.js

```diff
@@ -1733,3 +1733,43 @@ test('Should fail to invoke callNotFound inside a 404 handler', t => {
     t.is(res.payload, '404 Not Found')
   })
 })
+
+test('400 in case of bad url (pre find-my-way v2.2.0 was a 404)', t => {
```

Comment:
Should we document that behaviour anywhere?

Reply:
As a general rule, I think we should document as much as possible, but since this is an internal configuration which (currently) cannot be configured by the users, I'm not sure we need to document it, ad if so, where?